tests/auth_tests/test_mixins.py

```diff
@@ -0,0 +1,153 @@
+from django.contrib.auth import models
+from django.contrib.auth.mixins import (
+    LoginRequiredMixin, PermissionRequiredMixin, UserPassesTestMixin,
+)
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import PermissionDenied
+from django.http import HttpResponse
+from django.test import RequestFactory, TestCase
+from django.views.generic import View
+
+from .test_views import AuthViewsTestCase
+
+
+class EmptyResponseView(View):
+
```

Comment:
do we need a blank line before each method? seems like it just makes things longer.